script.js

```diff
@@ -232,13 +232,20 @@ const form = document.querySelector('#form');
 
 function contactValidator(event) {
   const email = document.querySelector('#email').value;
+  const name = document.querySelector('#name').value;
+  const message = document.querySelector('#message').value;
+
   const emailValidation = /[A-Z]/.test(email);
 
   if (emailValidation) {
     emailError.textContent = 'Email Not Sent⚠️, all the letters must be in lower case';
     event.preventDefault();
   } else {
     emailSent.textContent = 'Congratulations, Email sent 👏';
+    localStorage.setItem('name', name);
+    localStorage.setItem('message', message);
+    localStorage.setItem('email', email);
   }
 }
```

Comment:
- [ ] Nice work saving the data in the browser :clap: However it would be good if you stored the form data for all inputs in a single object as requested in the project requirement.
E.g: 
```sh
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```